warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ContentIndexingColumnBasedHandler.java

```diff
@@ -369,16 +369,16 @@ protected void tokenizeField(final Analyzer a, final NormalizedContentInterface
             // Track amount of time we've spent tokenizing this document,
             // at the least we will use this for metrics, we could also use
             // this to halt indexing if we exceed a certain threshold
-            int heartBeatCount = HeartBeatThread.counter;
+            int heartBeatCount = HeartBeatThread.getCounter();
             int tokenizerBeats = 0;
             long start = System.currentTimeMillis();
             
             tokenizerTimeWarned = false;
             
             while (true) {
-                if (heartBeatCount != HeartBeatThread.counter) {
-                    tokenizerBeats += HeartBeatThread.counter - heartBeatCount;
-                    heartBeatCount = HeartBeatThread.counter;
+                if (heartBeatCount != HeartBeatThread.getCounter()) {
+                    tokenizerBeats += HeartBeatThread.getCounter() - heartBeatCount;
+                    heartBeatCount = HeartBeatThread.getCounter();
```

Comment:
I think this would be more accurate.  HeartBeatThread.getCounter() could return a different value every time it's called.

```suggestion
                int currentHeartBeatCount = HeartBeatThread.getCounter()
                if (heartBeatCount != currentHeartBeatCount) {
                    tokenizerBeats += currentHeartBeatCount - heartBeatCount;
                    heartBeatCount = currentHeartBeatCount;
```

Reply:
fixed